test.disabled/ejabberd_tests/tests/mod_http_notification_SUITE.erl

```diff
@@ -96,6 +100,35 @@ process_notification(Req, Pid) ->
 %%%===================================================================
 %%% offline tests
 %%%===================================================================
+proper_http_message_encode_decode(Config) ->
+    Receiver= <<"bob">>,
+    Message = <<"Hi Test!&escape=Hello">>,
+    Sender = <<"alice">>,
+    Server = escalus_users:get_host(Config, alice),
+    BobJid = escalus_users:get_jid(Config, bob),
+
+    escalus:story(Config, [{alice, 1}],
+        fun(Alice) -> escalus:send(Alice, escalus_stanza:chat_to(BobJid, Message)) end),
+
+    Bob = login_send_presence(Config, bob),
```

Comment:
Is `Bob` used anywhere?

Reply:
Bob is used for receiving the Http request sent by Alice.

Reply:
I don't think that logged in XMPP client is required to receive HTTP notification?